client/app/scripts/utils/metric-utils.js

```diff
@@ -2,24 +2,19 @@ import { includes } from 'lodash';
 import { scaleLog } from 'd3-scale';
 import React from 'react';
 
-import { NODE_BASE_SIZE, NODE_SHAPE_DOT_RADIUS } from '../constants/styles';
 import { formatMetricSvg } from './string-utils';
 import { colors } from './color-utils';
 
-export function getClipPathDefinition(clipId, height) {
+export function getClipPathDefinition(clipId, height, radius) {
   return (
     <defs>
-      <clipPath id={clipId} transform={`scale(${NODE_BASE_SIZE})`}>
+      <clipPath id={clipId} transform={`scale(${4 * radius})`}>
```

Comment:
Meh, due to the way `height` is calculated in `getMetricValue`, this value has to be twice what you'd normally expect. Let me see if there's a quick fix to that, otherwise I'll just add a comment.